README.md

```diff
@@ -36,6 +36,7 @@ Just run:
 rails new store --skip-javascript
 cd store
 bundle add solidus_core solidus_backend solidus_api solidus_sample
+bundle add rspec-rails "~> 4.0.0.beta3"
```

Comment:
If I got it right, we're adding the version lock`~> 4.0.0.beta3` because that's what solidus_dev_support does. I'm wondering if this is going to become a maintenance burden since it seems we'll need to keep the 2 gem specifications in sync. Is that correct?

Reply:
@spaghetticode Yes, currently we need to add the version lock `~> 4.0.0.beta3` in order to allow `solidus_dev_support` to be installed. I believe that if we keep the version constraints of rspec-rails and puma loose in `solidus_dev_support`, then we won't have to constantly update these 2 gem specifications. I'll update https://github.com/solidusio/solidus_dev_support/pull/178 to loosen the constraint of rspec-rails as well.

Reply:
Now that https://github.com/solidusio/solidus_dev_support/pull/178 is merged do we still need that specificity in the version?

Reply:
@kennyadsl Yes, because the `solidus_dev_support` change still needs to be released. rspec-rails and puma are limited by the version of `solidus_dev_support` produced by RspecGenerator (see https://github.com/nebulab/solidus_starter_frontend/pull/180/files#diff-473c3b95b5f6d381b98b0f0d423c038ff4e0cab4d078c9fd2fdf26c489771dd6R19). If we point `solidus_dev_support` in that generator to the gem's master branch, it would make RspecGenerator (and `solidus_starter_frontend`) not ready for release.

Reply:
https://rubygems.org/gems/solidus_dev_support/versions/2.5.1 just released!

Reply:
Awesome! I'll update the development dependencies on this PR. TODO.